<launch>

  <!-- Launch rviz.launch before running this demo to see visalizations -->

  <!-- Debug -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <!-- PANDA -->
  <include file="$(find moveit_grasps)/launch/load_panda.launch">
  </include>

  <!-- Start the demo -->
  <node name="moveit_grasps_demo" launch-prefix="$(arg launch_prefix)" pkg="moveit_grasps"
  type="moveit_grasps_grasp_poses_visualizer_demo" output="screen" args="">
    <param name="ee_group_name" value="hand"/>
    <param name="planning_group_name" value="panda_arm"/>
    <rosparam command="load" file="$(find moveit_grasps)/config_robot/panda_grasp_data.yaml"/>
    <rosparam command="load" file="$(find moveit_grasps)/config/moveit_grasps_config.yaml"/>
  </node>

</launch>
